packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1467,9 +1471,7 @@ export class AccountHandler {
 
     const { uid, email } = request.auth.credentials;
 
-    let subscriptions: Awaited<
-      ReturnType<StripeHelper['subscriptionsToResponse']>
-    > = [];
+    let subscriptions: Awaited<MozillaSubscription[]> = [];
```

Comment:
This might be the method to modify for [FXA-4105](https://mozilla-hub.atlassian.net/browse/FXA-4105)? I actually don't see any GETs to the auth server when I look at the Network Monitor for FxA Settings. But that's probably because FxA Settings uses GraphQL.